src/app/views/apps/dashboard-apps-list.controller.coffee

```diff
@@ -104,11 +104,19 @@ angular.module 'mnoEnterpriseAngular'
       # Post-Initialization
       #====================================
       $scope.$watch MnoeOrganizations.getSelectedId, (val) ->
-        if val?
-          $scope.isLoading = true
-          MnoeAppInstances.getAppInstances().then(
-            ->
-              $scope.isLoading = false
-              $scope.apps = MnoeAppInstances.appInstances
-          )
+        $scope.isLoading = true
+        ImpacConfigSvc.getOrganizations().then(
+          (resp) ->
+            selectedOrg = _.find(resp.organizations, { id: parseInt(val) })
```

Comment:
FYI `parseInt` will need to be removed in branch `2.0`

Reply:
ok - what do you use instead?

Reply:
in `2.0`, ids are typed as string